rmw_connext_shared_cpp/include/rmw_connext_shared_cpp/wait.hpp

```diff
@@ -347,7 +447,6 @@ wait(
       }
       DDS::ReadCondition * read_condition = client_info->read_condition_;
       if (!read_condition) {
-        RMW_SET_ERROR_MSG("read condition handle is null");
```

Comment:
Why remove this?

Reply:
looks like this was unintentional, added back